package.json

```
@@ -13,10 +13,11 @@
     "babel-core": "^4.7.16",
     "babel-loader": "^4.2.0",
     "babelify": "^5.0.4",
+    "browserify": "^9.0.3",
     "concat-stream": "^1.4.7",
     "cuid": "^1.2.4",
     "debug": "^2.1.3",
-    "flummox": "^3.1.2",
+    "flummox": "^3.3.1",
```

Errors were swallowed up until 3.3.1.